app/src/main/java/org/novinger/jason/demoreader/ListActivity.java

```diff
@@ -1,47 +1,70 @@
 package org.novinger.jason.demoreader;
 
+import android.os.StrictMode;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.Menu;
 import android.view.MenuItem;
 import android.view.View;
+import android.view.ViewPropertyAnimator;
 import android.widget.AdapterView;
 import android.widget.ArrayAdapter;
 import android.widget.ListView;
 
+import com.google.gson.Gson;
+
+import org.novinger.jason.demoreader.datamodels.Article;
+
+import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Collections;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
 
 public class ListActivity extends AppCompatActivity {
 
+    private final static int qty = 50;
+    private final static String feedUrl = "http://www.dailydot.com/api/v1/content/article/";
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_list);
 
+        StrictMode.ThreadPolicy policy = new StrictMode.ThreadPolicy.Builder().permitAll().build();
+        StrictMode.setThreadPolicy(policy);
+        Article[] articles;
+        try {
+            articles = getFeedItems(getFeedUrl());
+        } catch (IOException ex) {
+            articles = new Article[0];
+            Log.d("Exception", ex.toString());
+        }
+
         final ListView listview = (ListView) findViewById(R.id.listview);
-        String[] values = new String[] {
-                "Jason", "Cindy", "Hannah", "Leah", "Kirsten", "Matt", "Elle", "Trevor", "Alex",
-                "Eli", "Danika", "Lori", "Wyatt", "Chad"
-        };
-        final ArrayList<String> list = new ArrayList<String>();
-        Collections.addAll(list, values);
+        final ArrayList<String> list = new ArrayList<>();
+        for (Article article : articles) {
+            list.add(article.toString());
+        }
 
         final ArrayAdapter adapter = new ArrayAdapter(this, android.R.layout.simple_list_item_1, list);
```

Comment:
Yep, I just got excited about seeing things on the screen. [`RecyclerView`](https://github.com/jnovinger/DemoReader/pull/2) is coming. :D